front/src/stats/Courses/Overview/Overview.jsx

```diff
@@ -80,114 +90,152 @@ const Overview = (props) => {
             <Breadcrumb.Item
               href="#"
               active
-            >{`Tiempos ${'cursos'}`}</Breadcrumb.Item>
+            >{`Resumen ${state.current}`}</Breadcrumb.Item>
           </Breadcrumb>
         </Col>
       </Row>
       <Row>
         <Col>
           <h2 className="content-header">
-            Página Principal: curso de prueba con un nombre muy largo que es
-            claramente del cms
+            Curso:{' '}
+            {state.allowed ? (
+              state.courseName === '' ? (
+                <Fragment>
+                  {state.current}{' '}
+                  <Spinner animation="border" variant="primary" />
+                </Fragment>
+              ) : (
+                state.courseName
+              )
+            ) : (
+              <Fragment>Sin información</Fragment>
+            )}{' '}
           </h2>
         </Col>
       </Row>
-      <Row>
-        <Col md={4}>
-          <CountBox
-            end={3325}
-            duration={2.75}
-            image={eyeIcon}
-            change={20}
-            caption={'Visitas totales'}
-          />
-        </Col>
-        <Col md={4}>
-          <CountBox
-            end={32}
-            duration={2.75}
-            image={userIcon}
-            change={20}
-            caption={'Usuarios Registrados'}
-          />
-        </Col>
-        <Col md={4}>
-          <CountBox
-            end={32}
-            duration={2.75}
-            image={clockIcon}
-            change={-40}
-            caption={'Minutos vistos'}
-            countUpProps={{
-              start: 10,
-              end: 3243.012,
-              duration: 2.75,
-              separator: '.',
-              decimals: 0,
-              decimal: ',',
+      {course.status === 'loading' && !dataLoaded ? (
+        <Row>
+          <Col style={{ textAlign: 'center' }}>
+            <Spinner animation="border" variant="primary" />
```

Comment:
I'd put the CountBoxes but with a loading spinner on each one

Reply:
Total visits and Users count boxes depends on the same call,  so if one is ready, both spinner would disappear